tests/unit/records/test_records_serializers_fields_export.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+import pytest
+
+from inspirehep.modules.records.serializers.fields_export import (
+    get_authors_with_role,
+    bibtex_document_type,
+    bibtex_type_and_fields,
+    get_year,
+    get_journal,
+    get_volume,
+    get_report_number,
+    get_type,
+    get_country_name_by_code,
+    get_author,
+)
+
+from inspire_schemas.utils import validate
+
+
+@pytest.fixture
+def test_record():
+    return {
+        "corporate_author": ["Corp A", "Corp B"],
+        "authors": [
+            {"full_name": "Kiritsis, Elias",
+             "inspire_roles": ["editor"]},
+            {"full_name": "Nitti, Francesco",
+             "inspire_roles": ["author"]},
+            {"full_name": "Pimenta, Leandro Silva"},
+        ],
+        "document_type": ["thesis"],
+        "thesis_info": {
+            "degree_type": "master",
+            "date": "1996-09",
+        },
+        "publication_info": [
+            {
+                "journal_title": "Rhys.Rev.",
+                "year": 2018,
+            },
+        ],
+        "titles": [{
+            "title": "Lorem Ipsum"
+        }],
+        "_collections": ["Literature"]
+    }
+
+
+def test_test_record_validity(test_record):
+    validate(test_record, 'hep')
+
+
+def test_get_author_by_role(test_record):
+    expected = ["Nitti, Francesco", "Pimenta, Leandro Silva"]
+    result = get_authors_with_role(test_record['authors'], 'author')
+    assert expected == result
+
+
+def test_get_editor_by_role(test_record):
+    expected = ["Kiritsis, Elias"]
+    result = get_authors_with_role(test_record.get('authors'), 'editor')
+    assert expected == result
+
+
+def test_bibtex_document_type(test_record):
+    expected = "mastersthesis"
+    result = bibtex_document_type("thesis", test_record)
+    assert expected == result
+
+
+def test_bibtex_document_type_prefers_article():
+    input = {"thesis_info": {"degree_type": "master"}, "document_type": ["thesis", "article"]}
```

Comment:
Some unit tests use local data, some others use a fixture. Let's be consistent! Having self-contained tests has the advantage of giving very clear failure messages, while having common fixtures avoids repetition. In this case we shouldn't have repetition anyway (all tests are testing different fields), so I'd lean towards local data everywhere.